q2studio/api/workspace.py

```diff
@@ -75,8 +76,12 @@ def get_artifacts():
 @fail_gracefully
 def create_artifact():
     request_body = request.get_json()
+    _format = None
+    if request_body['source_format']:
+        _format = request_body['source_format']
```

Comment:
`source_format` key exists but the value in the JSON object is `''` not `null`.

```
def create_artifact():
    request_body = request.get_json()
    print(request_body)
```

Prints:

`{'source_format': '', 'name': 'demo2222', 'path': '/Users/maxvonhippel/Desktop/demo/feature-table-v210.biom', 'type': 'FeatureTable[Frequency]'}`

The '``' value triggers `@fail_gracefully`.  I don't want to break the ability of the code to fail gracefully for null or empty values in the other (non-optional) params.  

I am going to try auto-changing `''` to `null` in the JSON and see if it breaks stuff/what happens.